pepper-apis/src/main/java/org/broadinstitute/ddp/service/UserService.java

```diff
@@ -0,0 +1,129 @@
+package org.broadinstitute.ddp.service;
+
+import org.broadinstitute.ddp.db.DBUtils;
+import org.broadinstitute.ddp.db.dao.ActivityInstanceDao;
+import org.broadinstitute.ddp.db.dao.DataExportDao;
+import org.broadinstitute.ddp.db.dao.JdbiEventConfigurationOccurrenceCounter;
+import org.broadinstitute.ddp.db.dao.JdbiMailAddress;
+import org.broadinstitute.ddp.db.dao.JdbiMedicalProvider;
+import org.broadinstitute.ddp.db.dao.JdbiQueuedEvent;
+import org.broadinstitute.ddp.db.dao.JdbiTempMailAddress;
+import org.broadinstitute.ddp.db.dao.JdbiUmbrellaStudy;
+import org.broadinstitute.ddp.db.dao.JdbiUser;
+import org.broadinstitute.ddp.db.dao.JdbiUserStudyEnrollment;
+import org.broadinstitute.ddp.db.dao.StudyGovernanceSql;
+import org.broadinstitute.ddp.db.dao.UserGovernanceDao;
+import org.broadinstitute.ddp.db.dao.UserProfileDao;
+import org.broadinstitute.ddp.db.dto.StudyDto;
+import org.broadinstitute.ddp.elastic.ElasticSearchIndexType;
+import org.broadinstitute.ddp.model.governance.Governance;
+import org.broadinstitute.ddp.model.governance.GrantedStudy;
+import org.broadinstitute.ddp.model.user.User;
+import org.broadinstitute.ddp.util.ConfigManager;
+import org.broadinstitute.ddp.util.ElasticsearchServiceUtil;
+import org.elasticsearch.action.bulk.BulkRequest;
+import org.elasticsearch.action.bulk.BulkResponse;
+import org.elasticsearch.action.delete.DeleteRequest;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.jdbi.v3.core.Handle;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * Entry-point for back-end operations related to users.
+ */
+public class UserService {
```

Comment:
I didn't see any tests for the new code. Did we forget to commit some files?

Reply:
The test is on its way.

Reply:
Created